dbschema/location.esdl

```diff
@@ -0,0 +1,28 @@
+module default {
+  abstract type Location extending Resource {
+    required name: str {
+      constraint exclusive;
+    }
+    required type: Location::Type;
+
+    isoAlpha3: Ethnologue::code {
+      constraint exclusive;
+    };
+
+    #TODO - links
+    #fundingAccountId: FundingAccount;
+    #defaultFieldRegionId: FieldRegion;
```

Comment:
I know these are just comments, but I feel like it's worth pointing out that this would be links to the object so
```suggestion
    #fundingAccount: FundingAccount;
    #defaultFieldRegion: FieldRegion;
```